includes/classes/sync/pull.php

```diff
@@ -105,13 +105,9 @@ public function saveContentToTable(
 
 		$data = [$column => $content];
 		$where = [$postIdColumn => $post_id];
-		$count = $wpdb->update($table,$data,$where); // false | int
+		$wpdb->update($table,$data,$where); // false | int
 
-		if($count === false || $count === 0){
-			$count = $wpdb->insert($table, array_merge($data, $where));
- 		}
-
-		return is_int($count);
+		return true;
```

Comment:
❓ Is it worth catching failure here and surfacing to the user?

This can fail if the given plugin exists but the post does not yet have an entry on that table.

Reply:
Yeah think it's a good idea, not to fail the whole process but more warn the user the pull wasn't successful is a good shout.